app/assets/javascripts/components/Navigation.es6

```diff
@@ -0,0 +1,53 @@
+(function() {
+  'use strict';
+
+  class Navigation extends PWPG.BaseComponent {
+    constructor($component) {
+      super($component);
+      this.init();
+    }
+
+    init() {
+      this.$trigger = $('.js-nav-toggler');
+      this.$target = $('.js-nav');
+      this.$nav = $('.l-page-nav');
+      this.mobileSize = '"mobile"';
+
+      this.setNavigationHiddenStatus();
+      this.bindEvents();
+    }
+
+    getPageSize() {
+      const el = window.getComputedStyle(this.$nav[0], ':after');
+
+      return el ? el.getPropertyValue('content') : this.mobileSize;
+    }
+
+    setNavigationHiddenStatus() {
+      this.$target.attr('aria-hidden', this.getPageSize() === this.mobileSize ? true : false);
```

Comment:
You could use `this.getPageSize().indexOf(this.mobileSize)` here to get around that double quote shenanigan